scrape.py

```diff
@@ -283,6 +297,11 @@ def main():
 
     for group in files:
         for build in group['builds']:
+            # Just use the specified data when the scraper cannot be used
+            if build.get('scraper') == 'noscraper':
+                build['link'] = build['url']
+                continue
```

Comment:
`continue` might be simple because after `else`:
```python
            build['date'] = parser.date
            build['link'] = parser.link
            build['size'] = parser.size
```